ocs_ci/utility/azure_utils.py

```diff
@@ -340,3 +340,16 @@ def stop_vm_instance(self, vm_name):
             self.cluster_resource_group, vm_name
         )
         result.wait()
+
+    def restart_vms_by_stop_and_start(self, node_names, timeout=540, wait=True):
```

Comment:
the default value for `timeout` here is different than the one in https://github.com/red-hat-storage/ocs-ci/pull/3607/files#diff-478e5b871f475f1e312384c0eeb216726f11da23e3d3ac4df80aa3ee5d92e441R1783

Is that intentional?

Reply:
No, it was not intentional. This method using methods `stop_nodes` and `start_nodes` which have been tested with a default timeout of 540s. Keeping the same default timeout value for this method too.  Also, the code has been rearranged after closing PR #3606